doc/website/advanced/usage-guide.md

```diff
@@ -79,3 +97,28 @@ count = 100
 ```
 
 When no config file is specified, a hard-coded version similar to [default config](../../../iceoryx_posh/etc/iceoryx/roudi_config_example.toml) will be used.
+
+### static configuration
```

Comment:
Section titles should start with a capitalised letter. That's what @ithier suggested.